cli/commands/project_sync.go

```diff
@@ -267,49 +292,65 @@ func (cmd *ProjectSync) WaitForUnisonContainer(containerName string, timeoutSeco
 
 // The local unison process is finished initializing when the log file exists
 // and has stopped growing in size
-func (cmd *ProjectSync) WaitForSyncInit(logFile string, timeoutSeconds int, syncWaitSeconds int) {
+func (cmd *ProjectSync) WaitForSyncInit(logFile string, workingDir string, timeoutSeconds int, syncWaitSeconds int) {
 	cmd.out.Info.Print("Waiting for initial sync detection")
 
-	var tempFile = fmt.Sprintf(".%s.tmp", logFile)
-	var timeoutLoopSleep = time.Duration(100) * time.Millisecond
-	// * 10 here because we loop once every 100 ms and we want to get to seconds
-	var timeoutLoops = timeoutSeconds * 10
-
-	// Create a temp file to cause a sync action
-	exec.Command("touch", tempFile).Run()
+	// The use of os.Stat below is not subject to our working directory configuration,
+	// so to ensure we can stat the log file we convert it to an absolute path.
+	if logFilePath, err := util.AbsJoin(workingDir, logFile); err != nil {
+		cmd.out.Info.Print(err.Error())
+	} else {
+		// Create a temp file to cause a sync action
+		var tempFile = ".rig-check-sync-start"
```

Comment:
Renamed this further to remove .tmp suffix and introduce rig in the name to clarify what it is and avoid potential sync ignores around not syncing tempfiles.